pywt/_dwt.py

```diff
@@ -401,3 +399,115 @@ def upcoef(part, coeffs, wavelet, level=1, take=0):
     if part not in 'ad':
         raise ValueError("Argument 1 must be 'a' or 'd', not '%s'." % part)
     return np.asarray(_upcoef(part == 'a', coeffs, wavelet, level, take))
+
+
+def pad(x, pad_widths, mode):
+    """Extend a 1D signal using a given boundary mode.
+
+    This function operates like ``numpy.pad`` but supports all signal extension
```

Comment:
given that you added intersphinx, I think you wanted this to be single backticks to create a link?

Reply:
That just gave me *numpy.pad* in italics. To get the intersphinx link to work I have to use ``:func:`numpy.pad```. Is there a configuration option that removes the need for explicitly adding `:func:`? What do you prefer?

Reply:
I know for SciPy it works, see for example [sweep_poly](http://scipy.github.io/devdocs/generated/scipy.signal.sweep_poly.html#scipy.signal.sweep_poly) linking to `numpy.poly1d` in its Notes section.

In the SciPy `conf.py` I don't immediately see what makes it work though, and that `conf.py` isn't exactly maintainable. So happy to use `:func:` here, simpler.

Reply:
> In the SciPy conf.py I don't immediately see what makes it work though

in a recent discussion at scikit-image, @NelleV indicated that it is the presence of
`default_role = "autolink"` in SciPy's `conf.py`
(We have not set a `default_role` in PyWavelets conf.py)